edx-olive/configuration-old
# SE-1021 Minor fixes to HOTG_ADDITIONAL_INSTANCE_TYPES

Updates comments and default value to match upstream master.

**JIRA tickets**: SE-1021 / CAM2-206 

**Discussions**: https://github.com/edx/configuration/pull/5348

**Testing instructions**:

1. Check that the changes made in https://github.com/edx/configuration/pull/5348 are reflected here.

**Author notes and concerns**:

1. Note that the value for `HOTG_ADDITIONAL_INSTANCE_TYPES` in the olive secure repo is a dict (`{}`) not a list (`[]`), so this change fixes the case where the default is not overridden.

**Reviewers**
